frontend/src/routes/+layout.svelte

```diff
@@ -24,8 +26,13 @@
     <div class="navbar-end">
       <div class="navbar-item">
         <div class="buttons">
-          <Signup />
-          <Login />
+          {#if $user === null}
```

Comment:
In future, can simply do {#if $user} logged in {:else} not logged in {/if}, but this is clear so all good :)

Reply:
agreed :) modified this too